services/usb-livesync-service-base.ts

```diff
@@ -39,7 +39,7 @@ class SyncBatch {
 					});
 				}
 				this.timer = null;
-			}, 500);
+			}, 250);
```

Comment:
Someone else has already figured it out: https://github.com/Microsoft/TypeScript/blob/master/src/compiler/tsc.ts#L487-L489